azure/datalake/store/transfer.py

```diff
@@ -447,18 +447,28 @@ def run(self, nthreads=None, monitor=True, before_start=None):
         before_start = None
         if monitor:
             self.monitor()
+            has_errors = False
+            error_list = list()
```

Comment:
Can also use `error_list = []` (unless you're overriding `list()` in tests or something 😉 ).